src/Microsoft.Azure.SignalRService/HubInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.SignalRService
```

Comment:
We may need a better name for this namespace.. this is not consistent with others

Reply:
How about Microsoft.Azure.SignalR.Service ?